Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+pelican = "==3.5.0"
+markdown = "*"
```

Comment:
Please specify a version.

Reply:
There is no `markdown` version specified in `requirements.txt`, so I am going with the latest version that is working without any issues, i.e. `2.6.11`